modules/reference/pages/rpk/rpk-cloud/rpk-cloud-auth-use.adoc

```diff
@@ -2,6 +2,8 @@
 
 Select the `rpk` cloud authentication (auth) to use.
 
+This swaps the current cloud auth to the specified cloud auth. If your current profile is a cloud profile, this unsets the current profile (since the auth is now different). If your current profile is for a self hosted cluster, the profile is kept.
```

Comment:
```suggestion
This swaps the current cloud authorization to the specified cloud authorization. If your current profile is a cloud profile, this unsets the current profile (because the authorization is now different). If your current profile is for a self-hosted cluster, the profile is kept.
```